### PR TITLE
Moving normalize, erase to functional_tensor.py

### PR DESCRIPTION
**Pull request by surgan12**
This is a follow up to #1466. 
Since, we have decided to shift the vertical to transforms via pytorch ops. This PR, moves existing such functions from functional.py to functional_tensor.py to ensure a unified setting.

### CODE REVIEW

**Comment by surgan12** [comment]
cc: @fmassa , what are your thoughts on this ?

File: torchvision/transforms/functional.py [hunk 1]
```diff
@@ -188,36 +188,6 @@ def to_pil_image(pic, mode=None):
     return Image.fromarray(npimg, mode=mode)
 
 
-def normalize(tensor, mean, std, inplace=False):
```
Comment (fmassa) [inline torchvision/transforms/functional.py]: This is a breaking change, because users of `torch.nn.functional.normalize` won't find it anymore.
At least, you should `from .functional_tensor import normalize, erase`

Comment (surgan12) [reply torchvision/transforms/functional.py]: Yeah, Got it.

Comment (fmassa) [reply torchvision/transforms/functional.py]: Given what I mentioned above, I'm not sure if there is a need to move `normalize` and `erase` to `functional_tensor.py`, because we don't have any implementation based on `PIL` anyway.

Thoughts?

Comment (surgan12) [reply torchvision/transforms/functional.py]: Yes but if we want to do something like dispatch from functional -> functional_tensor / functional -> functional_pil, then this might help in ensuring consistency, what do you think ?

Comment (surgan12) [reply torchvision/transforms/functional.py]: @fmassa, let me know if I am on same page as you?

Comment (fmassa) [reply torchvision/transforms/functional.py]: My thinking is that we don't need to move `normalize` nor `erase` to `functional_tensor.py`.
Users shouldn't need to worry about the existence of `functional_pil` nor `functional_tensor`, just `functional`.

Comment (surgan12) [reply torchvision/transforms/functional.py]: Sure, got it ! Thanks

**Comment by fmassa** [comment]
Thanks for the PR anyway!
